lib/seattleflu/db/cli/command/etl/enrollments.py

```diff
@@ -320,6 +326,20 @@ def upsert_encounter(db: DatabaseSession,
     return encounter
 
 
+def encounter_age(age_dict: dict) -> str:
+    """
+    Interpret the age of the individual at the time of encounter.
+    Converts age value from int to string to fit interval format. 
+    """
+    if not age_dict:
+        return None
```

Comment:
I think using `age_dict.get("value")` should handle lines 334-335.

Reply:
I kept this line because age_dict.get() errors out if the input is a None value